### PR TITLE
Added Imagefs inodes default settings in Eviction Hard

### PR DESCRIPTION
**Pull request by vaibhav2107**
Fixes : #44695

### CODE REVIEW

**Comment by vaibhav2107** [comment]
Hi @tengqm @kbhawkey 
Could you please have a look

File: content/en/docs/concepts/scheduling-eviction/node-pressure-eviction.md [hunk 1]
```diff
@@ -169,6 +169,7 @@ The kubelet has the following default hard eviction thresholds:
 - `nodefs.available<10%`
 - `imagefs.available<15%`
 - `nodefs.inodesFree<5%` (Linux nodes)
+- `imagefs.inodesFree<5%`
```
Comment (kannon92) [inline content/en/docs/concepts/scheduling-eviction/node-pressure-eviction.md:172]:
```suggestion
- `imagefs.inodesFree<5%` (Linux nodes)
```

Comment (vaibhav2107) [reply content/en/docs/concepts/scheduling-eviction/node-pressure-eviction.md]: @kannon92 thanks for the suggestion

**Comment by kannon92** [comment]
LGTM to me from technical side.

@sftim is this PR targeted to the right branch? I think maybe it should go to release-1.30 but not entirely sure on this one.

We did not end up cherry-picking this so its only valid for >= 1.30.

**Comment by sftim** [comment]
Thanks

/hold
Please change the base (ie, target) branch to _dev-1.30_ @vaibhav2107

**Comment by sftim** [comment]
OK to unhold once the target branch is correct.

**Comment by kannon92** [comment]
@vaibhav2107 could you address this point? https://github.com/kubernetes/website/pull/44696#issuecomment-1887898252

**Comment by drewhagen** [comment]
Hello @vaibhav2107  👋, A friendly reminder of timeline, this PR needs doc review complete by Doc Freeze on March 26th 18:00 PT. Please follow up by this date with setting the base branch to **_dev-1.30_** and have your review complete with `approved` and `lgtm` labels in order for it to be ready for this release.

Thanks!

**Comment by chanieljdan** [comment]
Hello @vaibhav2107 👋, A friendly reminder of timeline, this PR needs doc review complete by Doc Freeze on March 26th 18:00 PT. Looks like technical review is complete, but to meet the Docs Freeze this PR must have lgtm and approve labels applied, without any unaddressed comments or concerns from SIG Docs. 

Could we change the base (ie, target) branch to dev-1.30? Once done we can finalize docs review & approval.

**Comment by vaibhav2107** [comment]
Hi, I have changed the target branch from base to dev-1.30, you can review the PR and let me know if something else has to be done

**Comment by vaibhav2107** [comment]
@chanieljdan Thanks for the help

**Comment by drewhagen** [comment]
Given kannon92's comment

> LGTM to me from technical side.
> 
> @sftim is this PR targeted to the right branch? I think maybe it should go to release-1.30 but not entirely sure on this one.
> 
> We did not end up cherry-picking this so its only valid for >= 1.30.

/lgtm

**Comment by chanieljdan** [comment]
Thanks @vaibhav2107!

@kubernetes/sig-docs-en-owners does this PR look good for approval?

**Comment by vaibhav2107** [comment]
Thanks @natalisucks @stmcginnis

**Pull request merged**